docs/policy_configuration.md

```diff
@@ -0,0 +1,160 @@
+# Policy Configuration
+
+## Preparation
+TO DO
+
+## Initial install and configuration
+TO DO
+
+## Day 2 configuration
+
+### Updates to ProvisioningRequest spec.templateParameters.clusterInstanceParameters
+
+A ProvisioningRequest can be edited to update:
+* the cluster labels and annotation
+```yaml
+spec:
+  templateParameters:
+    clusterInstanceParameters:
+      clusterName: cnfdg11
+      extraAnnotations:
+        ManagedCluster:
+          test: test <<< added as a Day 2 change
+      extraLabels:
+        ManagedCluster:
+          clustertemplate-a-policy: v1
+          test: test <<< added as a Day 2 change
+```
+* the node labels and annotations
+```yaml
+spec:
+  templateParameters:
+    clusterInstanceParameters:
+      clusterName: cnfdg11
+      nodes:
+      - bmcCredentialsName:
+          name: cnfdg11-bmc-secret
+        extraAnnotations:
+          BareMetalHost:
+            test: test <<< added as a Day 2 change
+        extraLabels:
+          BareMetalHost:
+            test: test <<< added as a Day 2 change
+```
+The `status.conditions` records the success/failure of the update.
+
+The cluster configuration goes to the `ManagedCluster` CR and the nodes configuration to the corresponding `BMH`s, as expected.
+
+**Note**: ManagedCluster  and node extra labels&annotations are the only fields that can be edited post installation. All the other fields are immutable and would be rejected by webhooks put in place by other operators for cluster installation resources (ex: `ClusterDeployment`)
+
+
+### Updates to ProvisioningRequest spec.templateParameters.clusterInstanceParameters
+
+These types of changes can be made under the ProvisioningRequest `spec.templateParameters` by updating the `policyTemplateParameters` entry, if it's present.
+```yaml
+apiVersion: o2ims.provisioning.oran.org/v1alpha1
+kind: ProvisioningRequest
+metadata:
+  finalizers:
+    - provisioningrequest.o2ims.provisioning.oran.org/finalizer
+  generation: 2
+  name: cnfdg11
+  resourceVersion: "99518665"
+  uid: 397a9bae-9483-417d-a808-3f60343760a4
+spec:
+  description: Provisioning request for basic SNO with sample ACM policies
+  name: Dev-main-SNO-Provisioning-sno-1
+  templateName: clustertemplate-a
+  templateParameters:
+    nodeClusterName: site-sno-1
+    oCloudSiteId: local-west-12345
+    policyTemplateParameters:
+      sriov-network-vlan-1: "111"
+      sriov-network-vlan-2: "222"
+```
+**Note:** Only policy configuration values that are exposed in the associated ClusterTemplate under `spec.templateParameterSchema > policyTemplateParameters` can be updated through the ProvisioningRequest.
```

Comment:
Should it be spec.templateParameterSchema.policyTemplateParameters?

Reply:
I didn't include `templateParameterSchema.policyTemplateParameters` because `policyTemplateParameters` is not an actual part of the `spec`, but part of the policy template schema and I wanted to make the distinction.